src/components/Table/GridTable.test.tsx

```diff
@@ -1655,6 +1655,51 @@ describe("GridTable", () => {
     expect(cell(r, 1, 1)).toHaveTextContent("foo2");
   });
 
+  it("getSelectedRows doesn't fire if data has not changed", async () => {
```

Comment:
This test is not related to the `rowApi.visibleChildren`, but while working on that I realized `RowState.row` was changing too often (after each `createRows` made new row literals), which in turn was causing `api.getSelectedRows` to be too chatty, so I went ahead and fixed it...

Granted, it kinda blew up the PR...